lib/openapi/revisions/fps-spec-1.5.0-20231130.json

```diff
@@ -0,0 +1,870 @@
+            },
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/VisitResponse"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request."
+          },
+          "401": {
+            "description": "Authorization failure due to invalid authentication credentials."
+          }
+        }
+      },
+      "get": {
+        "summary": "Retrieve data for a particular visit",
+        "description": "Returns data for a particular visit based on the visit ID.",
+        "operationId": "getVisitorDetails",
+        "tags": [
+          "Visits"
+        ],
+        "security": [
+          {
+            "ApiKeyAuth": []
+          }
+        ],
+        "parameters": [
+          {
+            "name": "visitId",
+            "in": "query",
+            "description": "Visit ID.",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the visit data.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/VisitDetails"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request"
+          },
+          "401": {
+            "description": "Authorization failure due to invalid authentication credentials."
+          }
+        }
+      }
+    },
+    "/api/ds/v1/visitors": {
+      "get": {
+        "summary": "Retrieve visitor data",
+        "description": "Retrieves all visit data for a given visitor using the supplied visitor ID.",
+        "operationId": "getAllVisitorData",
+        "tags": [
+          "Visitors"
+        ],
+        "security": [
+          {
+            "ApiKeyAuth": []
+          }
+        ],
+        "parameters": [
+          {
+            "name": "visitorId",
+            "in": "query",
+            "description": "Visitor ID.",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful retrieved visitor data.",
+            "headers": {
+              "x-photoniq-vid": {
+                "schema": {
+                  "type": "string"
+                },
+                "description": "The visitor ID.",
+                "example": "062b8104-6a71-4ac3-ac7f-6cd1ba6f2fa1"
+              }
+            },
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/AllVisitsResponse"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request"
```

Comment:
```suggestion
            "description": "Bad request."
```

Reply:
@joubert-macrometa , isn't there a longer version of this? Something about missing or malformed data?